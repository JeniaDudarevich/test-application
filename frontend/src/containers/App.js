import React from 'react';
import { Layout } from "antd";
import 'antd/dist/antd.css';
import HeaderContainer from "../components/HeaderContainer";
import { Route, Switch } from "react-router";
import VideosListPage from "../components/pages/VideosListPage";
import UploadVideoPage from "../components/pages/UploadVideoPage";

const { Header, Content } = Layout;

export default function App() {
  return (
    <Layout className="layout">
      <Header>
        <HeaderContainer />
      </Header>
      <Content>
        <Switch>
          <Route path="/upload" component={UploadVideoPage} />
          <Route path="/" component={VideosListPage} />
        </Switch>
      </Content>
    </Layout>
  )
}
