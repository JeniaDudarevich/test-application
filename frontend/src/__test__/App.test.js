import React from "react";
import {shallow} from "enzyme";
import App from "../containers/App";
import toJson from "enzyme-to-json";

describe("App", () => {
  it("renders without crashing", () => {
      const wrapper = shallow(<App />);
      expect(toJson(wrapper)).toMatchSnapshot();
  });
});
