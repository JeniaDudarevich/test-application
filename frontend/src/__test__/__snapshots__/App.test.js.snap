// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<Layout
  className="layout"
>
  <Header>
    <withRouter(HeaderContainer) />
  </Header>
  <Content>
    <Switch>
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "inject-with-VideoStore(UploadVideoPage)",
            "isMobxInjector": true,
            "render": [Function],
            "wrappedComponent": [Function],
          }
        }
        path="/upload"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "inject-with-VideoStore(VideosListPage)",
            "isMobxInjector": true,
            "render": [Function],
            "wrappedComponent": [Function],
          }
        }
        path="/"
      />
    </Switch>
  </Content>
</Layout>
`;
